website/python/brewmonitor/accessor/_app.py

```diff
@@ -1,8 +1,8 @@
-from flask import Blueprint
-
-accessor_bp = Blueprint(
-    'accessor',
-    __name__,
-    template_folder='../../../templates/accessor/',
-    url_prefix='/accessor'
+from flask import Blueprint
+
+accessor_bp = Blueprint(
+    'accessor',
+    __name__,
+    template_folder='../../../templates/accessor/',
+    url_prefix='/accessor'
 )
```

Comment:
Since we're modifying it let's add an empty line at the end

Reply:
Not sure what you mean. I can see an empty line at the end of this file.